bootstrap-orbit.sh

```diff
@@ -77,6 +77,36 @@ EOF
   fi # which dpkg-query
 fi # IGNORE_SYS_REQ
 
+function check_conan_version_sufficient() {
+  VERSION=$1
+  REQUIRED=$2
+
+  VERSION_MAJOR="$(echo "$VERSION" | cut -d'.' -f1)"
+  REQUIRED_MAJOR="$(echo "$REQUIRED" | cut -d'.' -f1)"
+  if [ "$REQUIRED_MAJOR" -gt "$VERSION_MAJOR" ]; then
+    return 1
+  fi
+  if [ "$REQUIRED_MAJOR" -lt "$VERSION_MAJOR" ]; then
+    return 0
+  fi
+
+  VERSION_MINOR="$(echo "$VERSION" | cut -d'.' -f2)"
+  REQUIRED_MINOR="$(echo "$REQUIRED" | cut -d'.' -f2)"
+  if [ "$REQUIRED_MINOR" -gt "$VERSION_MINOR" ]; then
+    return 1
+  fi
+  if [ "$REQUIRED_MINOR" -lt "$VERSION_MINOR" ]; then
+    return 0
+  fi
+
+  VERSION_PATCH="$(echo "$VERSION" | cut -d'.' -f3)"
+  REQUIRED_PATCH="$(echo "$REQUIRED" | cut -d'.' -f3)"
```

Comment:
ALL the variables should be declared `local`. If not they will be global variables.

Reply:
done